test/jasmine/tests/plot_api_test.js

```diff
@@ -348,7 +384,45 @@ describe('Test plot api', function() {
                 expect(gd.data[0].zmax).toBe(10);
 
                 expect(gd.data[0].contours).toEqual({start: 3, end: 9, size: 1});
-            });
+            })
+            .catch(fail)
+            .then(done);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6675409/23322635/ded66f7c-fab2-11e6-857e-a33954876102.png)

Not me :smile: